websites/L/Lazada/presence.ts

```diff
@@ -0,0 +1,68 @@
+const presence = new Presence({
+		clientId: "1201520071055515698",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let item: HTMLElement;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: 
+			"https://lzd-img-global.slatic.net/g/tps/tfs/TB1PApewFT7gK0jSZFpXXaTkpXa-200-200.png",
+		startTimestamp: browsingTimestamp,	
+	};
+
+	if (document.querySelector(".pdp-mod-product-badge-title")) { // PRODUCT
+		item = document.querySelector(".pdp-mod-product-badge-title");
+		presenceData.details = "Viewing Product:";
+		presenceData.state = item.textContent;
+		presenceData.largeImageKey = document.querySelector('meta[name="og:image"]').getAttribute("content") || "";
+	} else if (document.location.pathname.includes("/tag")) { // TAGS
```

Comment:
To cleanup the multiple uses of `document.location.*`, destructure the used properties:
```ts
const { pathname } = document.location;
...
if (pathname.includes(...)) {
  ...
}
```